snuba/datasets/factory.py

```diff
@@ -52,6 +53,7 @@ def get_dataset(name: str) -> Dataset:
         "transactions": TransactionsDataset,
         "outcomes": OutcomesDataset,
         "outcomes_raw": OutcomesRawDataset,
+        "session_events_raw": SessionEventsRawDataset,
```

Comment:
bear in mind, when you get close to commit this, the dataset must not be registered until the table is not present in production or snuba will fail to start.
The alternative is registering and adding in the list of disabled datasets in settings